ingestion_tools/scripts/importers/visualization_config.py

```diff
@@ -90,18 +90,21 @@ def _to_point_layer(
         name_prefix: str,
         color: str,
         resolution: tuple[float, float, float],
-        is_instance_segmentation: bool,
+        shape: str,
         **kwargs,
     ) -> dict[str, Any]:
-        return state_generator.generate_point_layer(
-            source_path,
-            f"{name_prefix} point",
-            self.config.https_prefix,
-            color=color,
-            scale=resolution,
-            is_visible=file_metadata.get("is_visualization_default"),
-            is_instance_segmentation=is_instance_segmentation,
-        )
+        args = {
+            "source": source_path,
+            "name": f"{name_prefix} point",
```

Comment:
@uermel  As the point and the orientend point are going to be rendered differently, should we distinguish that in the name as well?

Reply:
I agree conceptually, but would the names become too long if we add 'oriented'?

Reply:
On second thought 

`segmentation`
`orientedpoint`

have the same length, so I agree we should name properly.